crates/rust-analyzer/src/handlers/request.rs

```diff
@@ -1990,3 +2055,35 @@ fn to_url(path: VfsPath) -> Option<Url> {
     let str_path = path.as_os_str().to_str()?;
     Url::from_file_path(str_path).ok()
 }
+
+fn resource_ops_supported(config: &Config, kind: &ResourceOperationKind) -> anyhow::Result<()> {
```

Comment:
```suggestion
fn resource_ops_supported(config: &Config, kind: ResourceOperationKind) -> anyhow::Result<()> {
```
No need to take a reference here